client/services.go

```diff
@@ -58,3 +60,32 @@ func (client *Client) doMultiServiceAction(actionName string, services []string)
 	}
 	return client.doAsyncFull("POST", "/v1/services", nil, headers, bytes.NewBuffer(data))
 }
+
+// ServicesOptions are the filtering options for querying services.
+type ServicesOptions struct {
+	// Names is the list of service names to query for. If slice is nil or
+	// empty, fetch information for all services.
+	Names []string
+}
+
+// ServiceInfo holds status information for a single service.
+type ServiceInfo struct {
+	Name    string `json:"name"`    // service name
+	Default string `json:"default"` // default state: "start" or "stop"
+	Status  string `json:"status"`  // "active", "inactive", or "error"
```

Comment:
Can we keep up with the "Current" vs. "Default" terminology?  It feels a bit more inherently clear.

I also wonder if we should have some consts for the known values these fields may have, and then make them typed (still as strings, as that's easier to work with).

Reply:
Yeah, I wondered that too. I think Startup and Current like the CLI is most clear, so I've renamed to that across the board (manager and client). And I've added a type and consts in the Go client too (I had them only on the manager end).

My only request is that, for consistency, we change the YAML layer config to use `startup: enabled/disabled` instead of `default: start/stop`. (start/stop seem more like actions or commands than the startup state in any case). @niemeyer  Thoughts?

Reply:
Sounds sane. Thanks for making it all consistent.